README.md

```diff
@@ -92,6 +95,28 @@ vars:
   ]
 ```
 
+## Upgrading from 1.x to >=2.0.0
+If you were using the following variables:
+
+```yml
+vars:
+  dbt_artifacts_database: your_db
+  dbt_artifacts_schema: your_schema
+```
+
+You must now move these to the following model configs:
+
+```yml
+models:
+  ...
+  dbt_artifacts:
+    sources:
+      +database: your_db
+      +schema: your_schema
+```
+
+That's because the raw tables are now managed as dbt models.
```

Comment:
```suggestion
That's because the raw tables are now managed as dbt models. Be aware of any impact that [generate_database_name](https://docs.getdbt.com/docs/building-a-dbt-project/building-models/using-custom-databases#generate_database_name) and [generate_schema_name](https://docs.getdbt.com/docs/building-a-dbt-project/building-models/using-custom-schemas#how-does-dbt-generate-a-models-schema-name) macros may have on the final database/schema.
```